pg_search/src/postgres/customscan/pdbscan/mod.rs

```diff
@@ -901,6 +901,20 @@ impl CustomScan for PdbScan {
         // Remove the oid from the with_index object
         // This helps to reduce the variability of the explain output used in regression tests
         Self::cleanup_varibilities_from_tantivy_query(&mut json_value);
+
+        // Add a flag to indicate if the query is a full index scan
+        if state
+            .custom_state()
+            .base_search_query_input()
+            .contains_all_query()
+        {
+            json_value.as_object_mut().unwrap().shift_insert(
+                0,
+                "full_index_scan".to_string(),
+                true.into(),
+            );
+        }
```

Comment:
This feels like it deserves its own top-level field: i.e. `if ..contains_all_query() { explainer.add_str(..) }`

Reply:
Good idea. Done.